_data/groups.yml

```diff
@@ -0,0 +1,6 @@
+administrators:
+    color: "#FFAA00"
+    members:
+        - username: "ChemistryX"
+          role: "role goes here"
+          social: "social goes here"
```

Comment:
Change to my name, and how do we use social? Adding anything, email, twitter etc won't make it a link unfortunately. Maybe we don't bother with social? Because only I can add social stuff, but maybe it's only necessary to add social stuff for admins.

Reply:
Fixed